dotcom-rendering/src/components/ProductCardImage.tsx

```diff
@@ -0,0 +1,70 @@
+import { css } from '@emotion/react';
+import type { HTMLAttributes } from 'react';
+import type { ArticleFormat } from '../lib/articleFormat';
+import type { ProductImage } from '../types/content';
+import { Caption } from './Caption';
+import { Picture } from './Picture';
+
+interface ProductCardImageProps extends HTMLAttributes<HTMLDivElement> {
+	format: ArticleFormat;
+	image?: ProductImage;
+	url?: string;
+	label?: string;
```

Comment:
Is this used? I think at the moment, due to the spreading of props on the `div`, it's getting proxied through to that element. I don't think it's a valid attribute there, but TS seems unable to catch that, perhaps as a result of the spread?